pages/dkp/kommander/2.2/projects/applications/catalog-applications/index.md

```diff
@@ -9,7 +9,7 @@ Catalog applications are any third-party or open source applications that appear
 
 ## Upgrade catalog applications
 
-Before upgrading your catalog applications, verify the current and supported versions of the application.
+Before upgrading your catalog applications, verify the current and supported versions of the application. While platform applications must all have the same set and version of platform applications deployed, catalog applications are individual, so you can deploy and upgrade them individually for each project.
 
 <p class="message--note"><strong>NOTE: </strong>Catalog applications must be upgraded to the latest version BEFORE upgrading the Kubernetes version (or Konvoy version for managed Konvoy clusters) on attached clusters, due to the previous versions' incompatibility with Kubernetes 1.22.</p>
```

Comment:
Is this true for all the Catalog applications?

Reply:
yes, at least for 2.2, this affects all the catalog apps we provided in 2.1. this will not always be the case, so we'll likely update this messaging for 2.3

Reply:
👍